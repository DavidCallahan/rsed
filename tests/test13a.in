============================================================================
admarket/asx/common/test/NgramBenchmark.cpp     relative  time/iter  iters/s
============================================================================
StraightForward                                            810.43ns    1.23M
FastImpl                                         148.08%   547.30ns    1.83M
PrehashedFast                                    191.79%   422.56ns    2.37M
genHashNgram                                               161.87ns    6.18M
genHashNgramNew                                  215.83%    75.00ns   13.33M
============================================================================
