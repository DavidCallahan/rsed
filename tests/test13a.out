admarket/asx/common/test/NgramBenchmark.cpp,StraightForward,810.43,ns,1.23,M
admarket/asx/common/test/NgramBenchmark.cpp,FastImpl,547.30,ns,1.83,M
admarket/asx/common/test/NgramBenchmark.cpp,PrehashedFast,422.56,ns,2.37,M
admarket/asx/common/test/NgramBenchmark.cpp,genHashNgram,161.87,ns,6.18,M
admarket/asx/common/test/NgramBenchmark.cpp,genHashNgramNew,75.00,ns,13.33,M
